src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -39,63 +39,61 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		ppacToken: PPACToken? = nil,
 		completion: ((Result<Void, PPASError>) -> Void)? = nil
 	) {
-
+		
 		// Check if user has given his consent to collect data
 		if userDeclinedAnalyticsCollectionConsent {
 			Log.warning("Analytics submission abord due to missing users consent", log: .ppa)
 			completion?(.failure(.userConsentError))
 			return
 		}
-
+		
 		configurationProvider.appConfiguration().sink { [weak self] configuration in
-
+			
 			guard let self = self else {
 				Log.warning("Analytics submission abord due fail at creating strong self", log: .ppa)
 				completion?(.failure(.generalError))
 				return
 			}
-
+			
 			// Check configuration parameter
 			if Double.random(in: 0...1) > configuration.privacyPreservingAnalyticsParameters.common.probabilityToSubmit {
 				Log.warning("Analytics submission abord due to randomness", log: .ppa)
 				completion?(.failure(.probibilityError))
 				return
 			}
-
+			
 			// Last submission check
 			if self.submissionWithinLast23Hours {
 				Log.warning("Analytics submission abord due to submission last 23 hours", log: .ppa)
 				completion?(.failure(.submission23hoursError))
 				return
 			}
-
+			
 			// Onboarding check
 			if self.onboardingCompletedWithinLast24Hours {
 				Log.warning("Analytics submission abord due to onboarding completed last 24 hours", log: .ppa)
 				completion?(.failure(.onboardingError))
 				return
 			}
-
+			
 			// App Reset check
 			if self.appResetWithinLast24Hours {
 				Log.warning("Analytics submission abord due to app resetted last 24 hours", log: .ppa)
 				completion?(.failure(.appResetError))
 				return
 			}
-
+			
+			self.hoursSinceTestRegistrationToSubmitTestResultMetadata = configuration.privacyPreservingAnalyticsParameters.common.hoursSinceTestRegistrationToSubmitTestResultMetadata
+			self.probabilityToSubmitExposureWindows = configuration.privacyPreservingAnalyticsParameters.common.probabilityToSubmitExposureWindows
+			
 			if let token = ppacToken {
 				// Submit analytics data with injected ppac token
 				self.submitData(with: token, completion: completion)
 			} else {
 				self.generatePPACAndSubmitData(completion: completion)
 			}
-			
-			self.hoursSinceTestResultToSubmitKeySubmissionMetadata = configuration.privacyPreservingAnalyticsParameters.common.hoursSinceTestResultToSubmitKeySubmissionMetadata
```

Comment:
Please don't delete this. I need this for my task.

Reply:
sorry removed by accident with the merge conflicts, will return it